src/playbacks/html5_video/html5_video.js

```diff
@@ -261,19 +309,8 @@ export default class HTML5Video extends Playback {
     })
   }
 
-  checkBufferState(bufferedPos) {
-    var playbackPos = this.el.currentTime + 0.05; // 50 ms threshold
-    if (this.isPlaying() && playbackPos >= bufferedPos) {
-      this.trigger(Events.PLAYBACK_BUFFERING, this.name)
-      this.buffering = true
-    } else if (this.buffering) {
-      this.trigger(Events.PLAYBACK_BUFFERFULL, this.name)
-      this.buffering = false
-    }
-  }
-
   typeFor(src) {
-    return (src.indexOf('.m3u8') > 0) ? 'application/vnd.apple.mpegurl' : 'video/mp4'
```

Comment:
because `typeFor` was only called in `html5_video`, not `hls`, meaning it would always be `video/mp4`. With `hls` no type gets added as this was for the `<source>` tags which aren't used for the hls playback.